Jenkinsfile

```diff
@@ -14,35 +14,33 @@
 // the License.
 
 // DRYing out the Jenkinsfile...
-build_script = '''
-mkdir -p ${COUCHDB_IO_LOG_DIR}
 
-echo
-echo "Build CouchDB from tarball & test"
-builddir=$(mktemp -d)
-cd ${builddir}
+build_and_test = '''
+mkdir -p ${COUCHDB_IO_LOG_DIR}
+rm -rf build
+mkdir build
+cd build
 tar -xf ${WORKSPACE}/apache-couchdb-*.tar.gz
 cd apache-couchdb-*
 ./configure --with-curl
 make check || (build-aux/logfile-uploader.py && false)
+'''
 
-echo
-echo "Build CouchDB packages"
-cd ${builddir}
+make_packages = '''
 git clone https://github.com/apache/couchdb-pkg
+rm -rf couchdb
```

Comment:
Are you confident that this will not persist workspaces between runs? Leftover files from previous runs were why I had this in there previously; it caused problems with glob matching since multiple `apache-couchdb-*.tar.gz` files would be present down on L34-35.

Reply:
The contents of the workspace do get cleaned up, and I definitely ran into the globbing problem when I wasn't doing that correctly.